test/contrib/rails/redis_cache_test.rb

```diff
@@ -0,0 +1,100 @@
+ENV['DATADOG_TEST_REDIS_CACHE_HOST'] = '127.0.0.1'
+ENV['DATADOG_TEST_REDIS_CACHE_PORT'] = '46379'
```

Comment:
can we add these somewhere else like we do for databases? if ``REDIS_URL`` is working I think you can just set one.